### PR TITLE
fix(kibbeh): Update Dutch Translations

### PR DESCRIPTION
**Pull request by laurens-vrh**
Translated text on https://dogehouse.tv/download

### CODE REVIEW

File: kibbeh/public/locales/nl/translation.json [hunk 1]
```diff
@@ -119,9 +119,9 @@
       "starting": "Aan het downloaden...",
       "failed": "De automatische download is mislukt, probeer het later nog een keer",
       "visit_gh": "Bezoek Github Releases",
-      "prompt": "Click on the button below to start download",
+      "prompt": "Klik op de onderstaande knop om het downloaden te starten",
       "download_now": "Nu downloaden",
-      "download_for": "Download voor {{platform}}"
+      "download_for": "Download voor %platform%"
```
Comment (BronzW) [inline kibbeh/public/locales/nl/translation.json]: Why did you change that?

Comment (laurens-vrh) [reply kibbeh/public/locales/nl/translation.json]: The placeholder wasn't working. It just said "Download voor", I looked in the code, and it used %, not {{}}. In the new deployed test version it works and says "Download voor Windows" etc.

**Comment by Tchoukoualeu** [comment]
Could you please also update the following line
[https://github.com/benawad/dogehouse/blob/5e1e47b90320500a6fc3db123158028e164b17ac/kibbeh/public/locales/nl/translation.json#L262](https://github.com/benawad/dogehouse/blob/5e1e47b90320500a6fc3db123158028e164b17ac/kibbeh/public/locales/nl/translation.json#L262)

File: kibbeh/public/locales/nl/translation.json (hunk 1 above)
Comment (amitojsingh366) [reply kibbeh/public/locales/nl/translation.json]: @BronzW he did it correct, however @DoeHetGewoon  you're missing the `(%ext%)`
```suggestion
      "download_for": "Download voor %platform% (%ext%)"
```

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

**Comment by laurens-vrh** [comment]
@Tchoukoualeu That's already valid Dutch.

@amitojsingh366 I commited your suggestion

**Comment by amitojsingh366** [comment]
> 
> 
> @Tchoukoualeu That's already valid Dutch.
> 
> @amitojsingh366 I commited your suggestion

ooo the commit message you used for the suggestion is in wrong format... you will have to fix that

**Comment by laurens-vrh** [comment]
How can I change that message?

"fix(kibbeh): Update Dutch Translations" is in the right format, right?

**Comment by amitojsingh366** [comment]
> 
> 
> How can I change that message?
> 
> "fix(kibbeh): Update Dutch Translations" is in the right format, right?

yes, but first letter can't be caps so it should be: `fix(kibbeh): update Dutch Translations`

**Comment by laurens-vrh** [comment]
It's changed

**Comment by ermalsh** [comment]
Looks good now - ok to merge.

**Comment by amitojsingh366** [comment]
> Looks good now - ok to merge.

nop, the commit msgs aren't right

**Comment by laurens-vrh** [comment]
> > Looks good now - ok to merge.
> 
> nop, the commit msgs aren't right

The last one is right ..?
I don't know how to change past commit messages...

**Comment by amitojsingh366** [comment]
> > > Looks good now - ok to merge.
> > 
> > 
> > nop, the commit msgs aren't right
> 
> The last one is right ..?
> I don't know how to change past commit messages...

follow this https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message

**Comment by laurens-vrh** [comment]
Uhm... I honestly don't know what I did, can someone just close this and forget it ever existed?

**Comment by amitojsingh366** [comment]
> Uhm... I honestly don't know what I did, can someone just close this and forget it ever existed?

sure....
